ajax/libs/ant-design-pro/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "ant-design-pro",
+  "filename": "ant-design-pro.min.js",
+  "version": "1.1.0",
+  "description": "An enterprise-class UI design language and React-based implementation",
+  "license": "MIT",
+  "author": "偏右",
```

Comment:
Let's use English instead. I think `afc163 <afc163@gmail.com>` is good enough. See his other repository for example.